### DISCUSSION TITLE
還愁想做資料處理 但沒資料做嗎

### DISCUSSION THREAD

**Discussion by NcuJerryKao** [1101-GS3073-C/Python-Community-GS3073-C#496] [General]
https://archive.ics.uci.edu/ml/datasets.php
這網站中 包含超多超多個主題 可針對自己有興趣 有概念的主題做篩選
很不錯的是 他有標明Number of Instances及Number of Attributes 可視自己能力及需求做選擇

**Comment by lx3211** [comment]
好多呀
而且有某種既視感
好像看到了辨識機器人的驗證圖片來源

**Comment by manipulateed** [comment]
這個網站的元素也太多了吧🤤雖然暫時還不會運用這些資料，但總讓人躍躍欲試

**Comment by manipulateed** [comment]
這個網站的元素也太多了吧🤤雖然暫時還不會運用這些資料，但總讓人躍躍欲試

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習

**Comment by williamyen0623** [comment]
好多英文，看起來有點吃力
不過通常越難接觸的資源越能激發價值
相信用心使用 肯定能成為很棒的學習工具

**Comment by Ben6uo** [comment]
看來有更多資料可以方便我來整合一下了

**Comment by liangyinglly** [comment]
通識中心開的機器學習那門課老師就是請同學用這個網站的資料來做期末專題報告

**Reply by HsuChen88** [reply]
有了這個，這樣訓練模型可以有現成的素材來訓練了

**Comment by larry40305** [comment]
這裡面的資料非常豐富，很適合用來訓練模型

**Comment by bww0401** [comment]
感謝分享，有了這個就不苦惱怎麼訓練模型了!
